app/dashboard/templates/shared/issue_details.html

```diff
@@ -101,6 +101,16 @@
       </span>
     </label>
   </div>
+  <div class="col-12 col-md-4 btn-group-toggle">
+    <label class="btn btn-block btn-radio permissionless my-2 font-weight-bold py-3 px-4 text-left">
+      <input type="radio" name="permission_type" id="stake_required" value="stake_required" autocomplete="off">
+      {% trans "Stake Required" %}
+      <span class="d-block font-weight-normal">
+        <i class="fas fa-user-cog "></i>
+        {% trans "Any bounty hunter who wants to work on the issue will have to stake 10% of the value of the issue." %}
```

Comment:
maybe we can add a tooltip to include this full explanation next {% trans "Stake Required" %} and change the one displayed by a short version
`{% trans "Stake Required" %} <i class="far fa-info" data-toggle="tooltip" title="Any bounty hunter who wants to work on the issue will have to stake 10% of the value of the issue."></i> `